operations/app/terraform/modules/function_app/apps.tf

```diff
@@ -40,5 +40,15 @@ module "pdi_function_app" {
     "${each.value.mi_accountName}"        = each.value.mi_accountValue
     fhir_url                              = each.value.fhir_url
     "AzureWebJobs.convertToFhir.Disabled" = each.value.AzureWebJobs_convertToFhir_Disabled
+    SMARTYSTREETS_AUTH_ID                 = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/SmartyStreetsAuthID)"
+    SMARTYSTREETS_AUTH_TOKEN              = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/SmartyStreetsAuthToken)"
+    CONTAINER_URL                         = "https://${var.resource_prefix}datasa.blob.core.windows.net/bronze"
+    INTAKE_CONTAINER_URL                  = "https://${var.resource_prefix}datasa.blob.core.windows.net/bronze"
+    INTAKE_CONTAINER_PREFIX               = "decrypted/valid-messages"
```

Comment:
Should we change `INTAKE_CONTAINER_PREFIX` given the changes we're making to `IntakePipeline`? Or do we want to hold off on that until we can confirm that it runs in the cloud and make subsequent changes then?

Reply:
it's not merged yet - we can always add that to the PR for when that goes in